Makefile

```diff
@@ -4,7 +4,7 @@ BUILD_DATE=`date +%FT%T%z`
 PKG_LIST := $(shell go list ./... | grep -v /vendor/)
 DB_URL := postgres://postgres:postgres@localhost/miniflux_test?sslmode=disable
 
-.PHONY: linux linux-arm darwin build run clean test lint integration-test clean-integration-test
+.PHONY: linux linux-arm darwin miniflux build run clean test lint integration-test clean-integration-test
```

Comment:
It should be "freebsd" instead of "miniflux" here?

Reply:
yes, fixed in https://github.com/miniflux/miniflux/pull/86/commits/44da0e83c6101497b40df43b7d2840f5bc49ad55